config-model/src/test/java/com/yahoo/vespa/model/content/cluster/ClusterTest.java

```diff
@@ -57,6 +58,21 @@ public void requireThatSearchCoverageIsApplied() throws ParseException {
             assertEquals(0.23, config.dataset(0).higher_coverage_minsearchwait(), 1E-6);
             assertEquals(0.58, config.dataset(0).higher_coverage_maxsearchwait(), 1E-6);
             assertEquals(2, config.dataset(0).searchablecopies());
+            assertTrue(config.dataset(0).useroundrobinforfixedrow());
+        }
+    }
+
+    @Test
+    public void requireThatDispatchTuningIsApplied() throws ParseException {
+        ContentCluster cluster = newContentCluster(joinLines("<search>", "</search>"),
+                joinLines("<tuning>",
+                        "</tuning>"));
+        for (Dispatch tld : cluster.getSearch().getIndexed().getTLDs()) {
```

Comment:
Consider adding an assert for the number of dispatchers to ensure that we actually check something in the loop.

Reply:
Good point. It turned out that a few tests did not test anything.